x-pack/plugins/ml/public/datavisualizer/datavisualizer.html

```diff
@@ -29,63 +29,107 @@ <h1>{{indexPattern.title}}</h1>
+              i18n-default-message="all"
+            ></option>
           </select>
-          <span>documents per shard from a total of</span>
-          <span class="field-emphasis">{{ overallStats.totalCount }}</span>
-          <ng-pluralize count="overallStats.totalCount" when="{'1':'document', 'other':'documents'}">documents</ng-pluralize>
-          <i aria-hidden="true" class="fa fa-info-circle" tooltip-placement="right"
-            tooltip="Selecting a smaller sample size will reduce query run times and the load on the cluster."></i>
+          <span
+            i18n-id="xpack.ml.datavisualizer.documentsPerShardDescription"
+            i18n-default-message="documents per shard from a total of {wrappedTotalCount} {totalCount, plural, one {document} other {documents}}"
+            i18n-values="{
+              wrappedTotalCount: '<span class=\'field-emphasis\'>' + overallStats.totalCount + '</span>',
+              totalCount: overallStats.totalCount
+            }"
+          ></span>
+          <i
+            aria-hidden="true"
+            class="fa fa-info-circle"
+            tooltip-placement="right"
+            tooltip="{{ ::'xpack.ml.datavisualizer.documentsPerShardTooltip' | i18n: {
+              defaultMessage: 'Selecting a smaller sample size will reduce query run times and the load on the cluster.'
+            } }}"
+          ></i>
         </div>
 
       </div>
 
       <div class="kuiPanel kuiVerticalRhythm datavisualizer-panel card-panel">
       	<div class="euiText">
-          <h2 class="kuiSubTitle kuiVerticalRhythm">Metrics</h2>
+          <h2
+            class="kuiSubTitle kuiVerticalRhythm"
+            i18n-id="xpack.ml.datavisualizer.metricsTitle"
+            i18n-default-message="Metrics"
+          ></h2>
         </div>
 
         <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--responsive euiFlexGroup--wrap">
           <div class="field-sort-controls euiFlexItem euiFlexItem--flexGrowZero">
             <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--responsive">
               <div ng-if="showAllMetrics === false" class="field-count-label euiFlexItem euiFlexItem--flexGrowZero">
-                <span class="field-emphasis">{{ metricCards.length }}</span>
-                <ng-pluralize count="metricCards.length" when="{'1':'field exists', 'other':'fields exist'}">fields exist</ng-pluralize> in documents (
-                <span class="field-emphasis">{{ totalMetricFieldCount }}</span> in total)
+                <span
+                  i18n-id="xpack.ml.datavisualizer.showMetricsDescription"
+                  i18n-default-message="{wrappedMetricCardsCount} {metricCardsCount, plural, one {field exists} other {fields exist}} in documents ({totalMetricFieldCount} in total)"
+                  i18n-values="{
+                      wrappedMetricCardsCount: '<span class=\'field-emphasis\'>' + metricCards.length + '</span>',
```

Comment:
As above, the `<span>` tag should be dropped from here I think

![image](https://user-images.githubusercontent.com/7405507/49738752-156cbe00-fc88-11e8-9890-e0b49a7a8a0d.png)

Reply:
With the last **master** <code>\<span\></code>  tag render correctly
![image](https://user-images.githubusercontent.com/25555662/50217818-eb5b8000-039a-11e9-819c-135bf28a6663.png)